tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Grpc/GrpcDotNet/GrpcAspNetCoreServer/GrpcDotNetServerCommon.cs

```diff
@@ -44,8 +44,8 @@ internal static class GrpcDotNetServerCommon
                 }
 
                 var serviceName = tracer.DefaultServiceName ?? "grpc-server";
-
-                scope = tracer.StartActiveInternal(GrpcCommon.OperationName, parent: spanContext, tags: tags, serviceName: serviceName);
```

Comment:
Can we remove `GrpcCommon.OperationName` if it's not used anymore?

Reply:
It's used by the gRPC client integration, and I plan to remove it in the subsequent gRPC client PR 👍🏼